src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/EditTake.kt

```diff
@@ -0,0 +1,19 @@
+package org.wycliffeassociates.otter.common.domain.content
+
+import io.reactivex.Completable
+import org.wycliffeassociates.otter.common.data.model.Take
+import org.wycliffeassociates.otter.common.domain.plugins.LaunchPlugin
+import org.wycliffeassociates.otter.common.persistence.repositories.ITakeRepository
+import java.time.LocalDate
+
+class EditTake(
+        private val takeRepository: ITakeRepository,
+        private val launchPlugin: LaunchPlugin
+) {
+    fun editAndSave(take: Take): Completable {
```

Comment:
just keeping it named "edit" is probably fine; updating a last modified and stuff like that makes sense as part of that operation